README.md

```diff
@@ -5,3 +5,62 @@
 [![Build Status](https://travis-ci.org/lefticus/cpp_starter_project.svg?branch=master)](https://travis-ci.org/lefticus/cpp_starter_project)
 
 [![Build status](https://ci.appveyor.com/api/projects/status/ro4lbfoa7n0sy74c/branch/master?svg=true)](https://ci.appveyor.com/project/lefticus/cpp-starter-project/branch/master)
+
+
+## Getting Started
+
+
+    $ mkdir <project_directory_name>
```

Comment:
This stuff is repeat from the C++ Weekly video, but it's helpful to have it written out all in one place.
It would be completely unnecessary if this were a template repository.